game/theater/controlpoint.py

```diff
@@ -525,14 +532,60 @@ def get_carrier_group_name(self) -> Optional[str]:
                             if db.unit_type_from_name(u.type) in [
                                 Forrestal,
                                 Stennis,
+                                VINSON,
+                                CVN_71,
+                                CVN_72,
+                                CVN_73,
+                                CVN_75,
```

Comment:
AFAICT these cases don't happen. Where are these coming from?

Reply:
In preparation for #1787. Supercarriers are pretty to look at even if you don't have the module, so I'm future-proofing in case someone (such as myself) wants to use both supercarrier and non-supercarrier types in the same campaign.